Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityUnitTest.java

```diff
@@ -526,7 +535,7 @@ public void anyExceptionExitsActivityWithError() {
 
         assertExceptionIsReturned("sdk-error", new Exception("Error!"));
     }
-
+    
```

Comment:
We should add one more test to this class to verify that when using`disableAndroidPay`, Android Pay does not show up in `mSupportedPaymentMethodListView`'s adapter.

I can help write that test